test/functional/VM_Test/src/com/ibm/j9/monitor/tests/LockNurseryTest.java

```diff
@@ -72,8 +72,6 @@ public static void generateGarbageAndCollect(){
 		}
 		System.out.println("Finished generating garbage, requesting gc");
 		System.gc();
-		System.gc();
-		System.gc();
```

Comment:
We should watch this case too. I don't know exact reason we called `System.gc()` three times. We do have race condition in Metronome when we can not guarantee Global GC execution even we call it twice.

Reply:
This should retain two calls to System.gc()